website/docs/components/select/examples/loading_state.tsx

```diff
@@ -20,7 +20,7 @@ const Demo = () => (
       </Text>
       <div>
         <Select mt={2} mr='auto' id='loading-select-no-chevron'>
-          <Select.Trigger chevron={false} placeholder={<Spin size='xs' mx={4} />} />
+          <Select.Trigger chevron={false} placeholder={<Spin size='xs' mx={4} />} tabIndex={-1} />
```

Comment:
```suggestion
          <Select.Trigger chevron={false} placeholder={<Spin size='xs' mx={4} />} disabled />
```
I think it should just be disabled.